acceptance/tests/runs_puppet_facts.rb

```diff
@@ -0,0 +1,30 @@
+test_name "facter -p loads facts from puppet"
```

Comment:
I'm ok merging this and leaving that for later, when we actually have a workflow setup.